src/app/features/quest/quest-preview/quest-preview.component.html

```diff
@@ -98,6 +98,19 @@ <h5 id="title" *ngIf="!!service.title">{{ service.title }}</h5>
           </li>
         </ul>
       </ng-container>
+
```

Comment:
while you are here, can you also fix multiple ID having the same value?

`id="questStartIcon"`
`id="questEndIcon"`